packages/bits/src/lib/wizard-v2/wizard-horizontal/wizard-horizontal.component.html

```diff
@@ -1,5 +1,6 @@
 <div class="nui-wizard-horizontal-header-container">
-    <div class="nui-step-header-wrapper" *ngFor="let step of steps; let i = index; let isLast = last">
+    <div class="nui-step-header-wrapper" *ngFor="let step of steps; let i = index; let isFirst = first">
+        <div *ngIf="!isFirst" class="nui-wizard-horizontal-line" [ngClass]="{'nui-wizard-horizontal-line-completed': steps.get(i-1)?.completed && step.completed || selectedIndex === i}"></div>
```

Comment:
Hi, Vita. I think we should avoid adding a function call to the template (`steps.get()`) since function calls in templates create inefficiencies in change detection. I would be okay with using the existing getStepState function instead for now since we have [this](https://jira.solarwinds.com/browse/NUI-6129) ticket to come up with a better solution. That way, at least we know the function will be replaced with something more efficient soon.

Reply:
I created a `getStep` pipe for avoiding multiple function calls in template